launcher/agent/agent.go

```diff
@@ -90,6 +91,35 @@ func (a *agent) Attest(ctx context.Context) ([]byte, error) {
 	return resp.ClaimsToken, nil
 }
 
+func (a *agent) AttestCustom(ctx context.Context, aud string, nonce []string) ([]byte, error) {
```

Comment:
Oh this should probably take an `Opts` object, since we will definitely add things to this interface.

Reply:
+1

Reply:
Added